hknweb/utils.py

```diff
@@ -36,3 +37,20 @@ def get_rand_photo(width=400):
     with open(find("animal_photo_urls.txt")) as f:
         urls = f.readlines()
     return urls[randint(0, len(urls) - 1)].strip() + "?w=" + str(width)
+
+# date and time
+
+def get_semester(date):
```

Comment:
Is this used anywhere? If not, let's remove it

Reply:
I anticipated that it might be used soon, like if we add a semester field to events or support to filter by semester. Do we have to remove all unused utils?

Reply:
In both of those use cases, we would change the events database schema and add a `semester` field. Also, course surveys adds a `Semester` entity that can be utilized. 

In terms of anticipating use, we can remove it for now, and add it back when it is actually used (since it is already in Git). 

On the principle of keeping the codebase clean and as maintainable as possible, yes, we have to remove all unused utils.

Reply:
I decided to just add the semester to the detail page.

Reply:
LMAO ok solid, sounds good :))

Reply:
![image](https://user-images.githubusercontent.com/46059916/88473224-38d19480-ced0-11ea-8422-ab53b904c425.png)
Is it possible to capitalize `fall`?